euth_wagtail/templates/account/signup.html

```diff
@@ -5,109 +5,114 @@
-                {{ form.username|add_class:"form-control form-error" }}
-            {% else %}
-                {{ form.username|add_class:"form-control"}}
-            {% endif %}
-            {{ form.username.errors }}
-        </div>
-
-        <div class="form-group">
-            <label class="form-label col-form-label" for="{{ form.email.id_for_label }}">Email</label>
-            {% if form.email.errors %}
-                {{ form.email|add_class:"form-control form-error" }}
-            {% else %}
-                {{ form.email|add_class:"form-control"}}
-            {% endif %}
-            {{ form.email.errors }}
-        </div>
-
-        <div class="form-group">
-            <label class="form-label col-form-label" for="{{ form.password1.id_for_label }}">Password</label>
-            {% if form.password1.errors %}
-                {{ form.password1|add_class:"form-control form-error" }}
-            {% else %}
-                {{ form.password1|add_class:"form-control"}}
-            {% endif %}
-            {{ form.password1.errors }}
-        </div>
-
-        <div class="form-group">
-            <label class="form-label col-form-label" for="{{ form.password2.id_for_label }}">Password repeat</label>
-            {% if form.password2.errors %}
-                {{ form.password2|add_class:"form-control form-error" }}
-            {% else %}
-                {{ form.password2|add_class:"form-control"}}
-            {% endif %}
-            {{ form.password2.errors }}
-        </div>
-
-        <div class="form-group">
-            {% if form.terms_of_use.errors %}
-                {{ form.terms_of_use|add_class:"form-error" }}
-            {% else %}
-                {{ form.terms_of_use }}
-            {% endif %}
-            <label class="form-label col-form-label" for="{{ form.terms_of_use.id_for_label }}">
-                {% blocktrans with terms_of_use_url="/terms-of-use" %}
-                I accept the <a href="{{terms_of_use_url}}" target="blank">terms of use</a>.
-                {% endblocktrans %}
-            </label>
-            {{ form.terms_of_use.errors }}
-        </div>
-
-        <input type="hidden" id="{{ form.timezone.id_for_label }}" name="timezone" value="" />
-
-        <button type="submit" value="Register" class="btn btn-primary">{% trans "register" %}</button>
-    </form>
-
-    {% block socialaccounts %}
-    <div class="socialaccounts">
-        {% get_providers as socialaccount_providers %}
-        {% if socialaccount_providers %}
-        <p class="h6 text-uppercase sans-serif">{% trans "Login with social account" %}</h6>
-        <div class="socialaccount_ballot">
-            <ul class="socialaccount_providers">
-                {% include "socialaccount/snippets/provider_list.html" with process="login" %}
-            </ul>
-        </div>
-        {% endif %}
     </div>
-    {% endblock %}
-</div>
+    <div class="account__container--right">
+        {% include "./howto.html" %}
+    </div>
```

Comment:
I think the django way is to always add includes into the includes folder @Rineee is that right?

Reply:
Yes, thats right! And better not use ./, but specify the path with the whole folder structure after /templates, so if you put it into euth_wagtail/templates/account/includes, you would need 'account/includes/howto.html'. For includes, Django will look into all folders called template and look for the path.

Reply:
cool! thanks for explanation